### DISCUSSION TITLE
MagickImage.colorType backed by settings

### DISCUSSION THREAD

**Discussion by Gounlaf** [dlemstra/Magick.NET#1543] [General]
:wave:

Looking at the following lines:

https://github.com/dlemstra/Magick.NET/blob/91a18cbbc9c9d7f4f24957549becf42bc04f9b58/src/Magick.NET/MagickImage.cs#L605-L619

If getter is refering first to Settings, setter should update the value to Settings, no?

Regards.

**Comment by dlemstra** [comment]
I suspect there was a good reason to do it like this when I created the that code but I don't know why I have done it this way. I will need to investigate if I can change this.
